packages/slate-react/src/hooks/use-editor.tsx

```diff
@@ -2,6 +2,10 @@ import { createContext, useContext } from 'react'
 
 import { ReactEditor } from '../plugin/react-editor'
 
+/**
+ * @deprecated Use useSlateStatic instead.
+ */
+ 
 /**
  * A React context for sharing the editor object.
```

Comment:
The `@deprecated` flag should move just below ^this line here, in the same comment section. The above error "Delete `·`" means it doesn't like that there's a comment section that's not directly above a function or something like that.